packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -207,9 +210,9 @@ export const useArbTokenBridge = (
     addTransaction({
       type: 'deposit-l1',
       status: 'pending',
-      value: utils.formatEther(amount),
+      value: utils.formatUnits(amount, nativeCurrency.decimals),
       txID: tx.hash,
-      assetName: 'ETH',
+      assetName: nativeCurrency.symbol,
       assetType: AssetType.ETH,
```

Comment:
DumQ: Shouldn't this be nativeCurrency.AssetType depending on what the native currency is?

Reply:
Ideally, `assetType` will be either `Native` or `ERC-20` so we can rename `ETH` to `Native` in a next PR